index.bs

```diff
@@ -325,8 +321,16 @@ interface PictureInPictureWindow {
 };
 </pre>
 
-The {{PictureInPictureWindow}} is a simple proxy to get information such as
-dimensions about the current Picture-in-Picture window.
+A {{PictureInPictureWindow}} instance represents a <a>Picture-in-Picture
+window</a>. It has a |state| that is either |opened| or |closed|.
+
+The {{width}} attribute is the width in <a lt=px value>CSS pixels</a> of the
+<a>Picture-in-Picture window</a> associated with {{pictureInPictureElement}}.
```

Comment:
What about "The width attribute MUST return the width of the window in CSS pixels if the state is opened. Otherwise, it MUST return 0."

Note that I'm not sure how it's phrased in the CSS  spec for the "window width" so maybe copy some phrasing from there.

Reply:
Done.